apps/antalmanac/src/components/RightPane/CoursePane/SearchForm/SearchForm.tsx

```diff
@@ -106,7 +108,7 @@ const SearchForm = (props: { classes: ClassNameMap; toggleSearch: () => void })
                 </div>
             </form>
 
-            {(currentMonth === 8 || currentMonth === 9) && <HelpBox />}
+            {/*(currentMonth === 8 || currentMonth === 9) &&*/ <HelpBox />}
```

Comment:
Initialize `currentMonth` outside of this component so it doesn't ever have to be recalculated per re-render.
Create an array of months (also outside the component) that it should be active.
```suggestion
            const activeMonthIndices = [false, false, false, false, false, false, false, true, true, false, false, false]
            {/* activeMonthIndices[currentMonthIndex] && */ <HelpBox />}
```